app/src/app/[lng]/organization/[id]/account-settings/account/account-settings-tab.tsx

```diff
@@ -142,6 +145,54 @@ const AccountSettingsTab = ({ t }: { t: TFunction }) => {
       defaultValue="brand-settings"
     >
       <Tabs.List display="flex" flexDirection="column" gap="12px">
+        <Tabs.Trigger
+          value="account-details"
+          fontFamily="heading"
+          justifyContent={"left"}
+          letterSpacing={"wide"}
+          color="content.secondary"
+          lineHeight="20px"
+          fontStyle="normal"
+          fontSize="label.lg"
+          height="52px"
+          w={"223px"}
```

Comment:
Would be good not to hardcode the widths in case the translated text is longer in some languages